proposals/first-class-span-types.md

```diff
@@ -490,6 +476,14 @@ We see a couple of ways of solving this concern:
    On the other hand, that could affect partners abilities to take them up.
 2. We couple type conversions to a Compilation or look at other ways of providing the well-known type to it. This will take a bit of investigation.
 
+#### Answer
```

Comment:
Do we need to keep the open question (and answer) or is the update to the spec from that decision (copied below) sufficient?

> Any Span/ReadOnlySpan types are considered applicable for the conversion if they match by their fully-qualified name.

Reply:
I wasn't sure how this is usually done, but your suggestion makes sense to me, thanks.